File: Lib/Telerik/Examples/RssReader/BrowserForm.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using Telerik.WinControls;

namespace RssReader
{
    public partial class BrowserForm : Telerik.WinControls.UI.RadForm
    {
        public BrowserForm()
        {
            InitializeComponent();
        }

        public WebBrowser WebBrowser
        {
            get
            {
                return this.webBrowser1;
            }
        }
    }
}

File: Lib/Telerik/Examples/RssReader/BrowserForm.vb
Imports System.ComponentModel
Imports System.Text
Imports Telerik.WinControls

Namespace CS
	Partial Public Class BrowserForm
		Inherits Telerik.WinControls.UI.RadForm
		Public Sub New()
			InitializeComponent()
		End Sub

		Public ReadOnly Property WebBrowser() As WebBrowser
			Get
				Return Me.webBrowser1
			End Get
		End Property
	End Class
End Namespace
